docs/styles/custom-styles.css

```
@@ -0,0 +1,111 @@
+/*
+ * Custom Dokka theme matching the SVG-to-Compose website.
+ *
+ * Overrides Dokka's CSS variables from style.css, main.css, and ui-kit.
+ * Uses !important on key variables because Dokka's runtime-injected
+ * stylesheets load after custom stylesheets and override the cascade.
+ *
+ * Color palette sourced from SiteTheme.kt.
+ * Typography: Space Grotesk (UI) + JetBrains Mono (code).
+ */
+
+@import url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500;700&display=swap');
```

Stylelint (17.4.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500;700&display=swap'" (import-notation)

(import-notation)